src/main/java/ch/kalunight/zoe/repositories/BannedAccountRepository.java

```diff
@@ -0,0 +1,65 @@
+package ch.kalunight.zoe.repositories;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import ch.kalunight.zoe.model.dto.DTO;
+import net.rithms.riot.constant.Platform;
+
+public class BannedAccountRepository {
```

Comment:
Codacy found an issue: [The utility class name 'BannedAccountRepository' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=5905431)